prow/plugins/requiresig/requiresig.go

```diff
@@ -120,8 +120,10 @@ func hasSigLabel(labels []github.Label) bool {
 // (6) if the issue has only the sig label, do nothing
 // (7) if the issue has only the needs-sig label, do nothing
 func handle(log *logrus.Entry, ghc githubClient, cp commentPruner, ie *github.IssueEvent, mentionRe *regexp.Regexp) error {
+	// Ignore PRs, closed issues, and events that aren't new issues or sig label
+	// changes.
 	if ie.Issue.IsPullRequest() || ie.Issue.State == "closed" ||
-		(ie.Action != github.IssueActionOpened && ie.Action != github.IssueActionLabeled && ie.Action != github.IssueActionUnlabeled) {
+		(ie.Action != github.IssueActionOpened && (ie.Action != github.IssueActionLabeled && ie.Action != github.IssueActionUnlabeled || !isSigLabel(ie.Label.Name))) {
```

Comment:
nit `(a && b || c)` is probably clearer as `((a && b) || c)`, this if statement is a bit gnarly

Reply:
I moved part of the check into a func in an attempt to make things clearer.